Sources/Property.swift

```diff
@@ -50,6 +50,12 @@ extension MutablePropertyProtocol {
 	}
 }
 
+/// Represents a mutable property that can be composed.
```

Comment:
Can you document this a bit more explicitly? (i.e. explain what it means for a property to be composed.)